nginx/README.md

```diff
@@ -1,6 +1,6 @@
 # Nginx
 
-Docker image for [Nginx] version 1.22.1 and [FPM] version 8.1.15.
+Docker image for [Nginx] version 1.22.4 and [FPM] version 8.3.8.
```

Comment:
I get the following.

```
/var/www $ nginx -version
nginx version: nginx/1.24.0
```

and

```
/var/www $ php-fpm83 -version
PHP 8.3.8 (fpm-fcgi) (built: Jun  6 2024 17:04:22)
Copyright (c) The PHP Group
Zend Engine v4.3.8, Copyright (c) Zend Technologies
    with Zend OPcache v8.3.8, Copyright (c), by Zend Technologies
```

Reply:
Agreed. I think this was a typo on my part. Thank you for spotting that. Fixed now. 

* Reran `make bake && make up`
* `docker compose exec drupal bash`
* `nginx -version`
* `php-fpm83-version`

Same results as above.

Reply:
Resolving this comment thread as fixed.